pkgs/by-name/ba/bant/package.nix

```diff
@@ -41,7 +41,7 @@ buildBazelPackage rec {
     hash =
       {
         aarch64-linux = "sha256-8pLgn67kwVNdqhUXYsdi7OsArCZZmW55UPzUBlIyBbk=";
```

Comment:
Compiled on a Pi5: The aarch64-linux hash is now `sha256-M6LMaqPli71YvJS/4iwvowCyVaf+qe8WSICR3CgdU34=`